wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt

```diff
@@ -420,6 +420,42 @@ class PrunerTest {
     assertThat((pruned.getType("SomeFieldOptions") as MessageType).field("c")).isNull()
   }
 
+  @Test @Ignore("Need to address #1327.")
+  fun includeExtensionOfOptionPrunesOtherExtensionFields() {
+    val schema = RepoBuilder()
+        .add("lecture.proto", """
+             |package squareup.options.lecture;
+             |
+             |import "google/protobuf/descriptor.proto";
+             |
+             |extend google.protobuf.FieldOptions {
+             |  optional bool relevant = 22301;
+             |  optional bool unrelevant = 22302;
+             |  optional bool unused = 22303;
+             |}
+             |
+             |message Lecture {
+             |  optional string title = 1 [(squareup.options.lecture.relevant) = true];
+             |  optional string content = 2 [(squareup.options.lecture.unrelevant) = true];
+             |}
+             """.trimMargin())
+        .schema()
+    val pruned = schema.prune(PruningRules.Builder()
+        .include("squareup.options.lecture.Lecture")
+        .include("google.protobuf.FieldOptions#squareup.options.lecture.relevant")
```

Comment:
we do that for fields like here https://github.com/square/wire/blob/master/wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt#L471-L496

Is it fair to expect the same to happen? Otherwise, it would be really hard to whitelist the options we would wanna consume in a big code base.

Reply:
I think it shouldn't be needed. Including `Lecture` should automatically include all options it uses.

Reply:
Agreed with Egor that this include shouldn’t do anything? I’m trying to think of what the right syntax is to include/exclude a set of extensions